swagger_spec_validator/validator20.py

```diff
@@ -285,10 +285,10 @@ def validate_unresolvable_path_params(path_name, path_params):
 
     :raises: :py:class:`swagger_spec_validator.SwaggerValidationError`
     """
-    msg = "Path Parameter used is not defined"
     for path in get_path_params_from_url(path_name):
         if path not in path_params:
-            raise SwaggerValidationError("%s: %s" % (msg, path))
+            msg = "Path parameter {!s} used is not documented on {!s}".format(path, path_name)
```

Comment:
Please use ``msg = "Path parameter '{}' used is not documented on '{}'".format(path, path_name)``. This will quote the two parameters and will be python2 and python3 friendly. (NOTE: you don't need to specify the conversion format ``!s`` since the parameters are already strings)